DllImportGenerator/DllImportGenerator/ManualTypeMarshallingAnalyzer.cs

```diff
@@ -0,0 +1,342 @@
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+#nullable enable
+
+namespace Microsoft.Interop
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class ManualTypeMarshallingAnalyzer : DiagnosticAnalyzer
+    {
+        private const string DiagnosticIdPrefix = "INTEROPGEN";
+        private const string Category = "Interoperability";
+
+        public readonly static DiagnosticDescriptor BlittableTypeMustBeBlittableRule =
+            new DiagnosticDescriptor(
+                "INTEROPGEN001",
+                "BlittableTypeMustBeBlittable",
+                "Type '{0}' is marked with BlittableTypeAttribute but is not blittable.",
```

Comment:
`BlittableTypeAttribute` -> `'BlittableTypeAttribute'`

Single quote around code/types was the norm for messages in the roslyn-analyzers repo.